Realm/RLMObjectStore.mm

```diff
@@ -289,18 +289,23 @@ static inline void RLMVerifyInWriteTransaction(RLMRealm *realm) {
 
 static inline void RLMInitializeSwiftListAccessor(RLMObjectBase *object) {
     // switch List<> properties to linkviews from standalone arrays
-    if ([object isKindOfClass:NSClassFromString(@"RealmSwift.Object")]) {
+    static Class s_swiftObjectClass = NSClassFromString(@"RealmSwift.Object");
```

Comment:
This function is very deeply nested and would benefit from some early returns/continues.